### DISCUSSION TITLE
Issues with resizing when using egui-wgpu

### DISCUSSION THREAD

**Discussion by IceSentry** [emilk/egui#1757] [Q&A] [answered]
I recently managed to integrate egui in my custom wgpu renderer. I'm using the main branch for egui/egui-wgpu/egui-winit. Everything seems to work, I can drag windows, collapse windows, use sliders and click buttons. The only thing that doesn't seem to work is resizing windows. I essentially have this egui window and everything except resizing works

```rust
    egui::Window::new("Hello title")
        .resizable(true)
        .collapsible(true)
        .show(&ctx, |ui| {
            ui.label("Hello label");
            if ui.button("test").clicked() {
                log::info!("click");
            }

            ui.add(egui::Slider::new(&mut *local, 0.0..=1.0));
        });
```
Do I need some sort of special handling for that? I'm pretty much just piping everything to the libraries and considering everything else works I assume I must be missing something. 

Is there a full example somewhere of an integration using those 3 libraries?

**Comment by emilk** [comment, marked as answer]
There is no widget in your window that can expand - they all have fixed size. If you want a window to expand, add something to it that can expand. See https://docs.rs/egui/latest/egui/#auto-sizing-panels-and-windows

**Reply by IceSentry** [reply]
Ah, right makes sense, I didn't realize that was how resizing works thanks!
